include/RAJA/policy/cuda/MemUtils_CUDA.hpp

```diff
@@ -211,6 +213,16 @@ void launch(const void* func, cuda_dim_t gridDim, cuda_dim_t blockDim, void** ar
   launch(stream);
 }
 
+RAJA_INLINE
+void launch(const void* func, cuda_dim_t gridDim, cuda_dim_t blockDim, void** args, size_t shmem,
+            cudaStream_t stream, const char *name)
```

Comment:
You could have one `launch` that has `name` default to `nullptr` and in the body:

```
if (name) nvtxRangePushA(name);
cudaErrchk(cudaLaunchKernel(func, gridDim, blockDim, args, shmem, stream));
if (name) nvtxRangePop();
```